src/public/app/widgets/dialogs/add_link.js

```diff
@@ -9,13 +9,9 @@ const TPL = `
     <div class="modal-dialog modal-lg" style="max-width: 1000px" role="document">
         <div class="modal-content">
             <div class="modal-header">
-                <h5 class="modal-title mr-auto">${t('add_link.add_link')}</h5>
-
+                <h5 class="modal-title flex-grow-1">${t('add_link.add_link')}</h5>
                 <button type="button" class="help-button" title="${t('add_link.help_on_links')}" data-help-page="links.html">?</button>
-
-                <button type="button" class="close" data-dismiss="modal" aria-label="${t('add_link.close')}" style="margin-left: 0 !important;">
-                    <span aria-hidden="true">&times;</span>
-                </button>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="${t('add_link.close')}"></button>
```

Comment:
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/583b47e8-bc24-4feb-9369-217676b7c9c3">

Background regression for input.

Reply:
Is this in dark mode? Do you think the input background color should be the same as the modal's background color? I'm not very sure about the design strategy for dark mode, but for me using white for input background color feels fine.

Reply:
@JYC333 , all the screenshots I've taken are in the official dark theme.

Here's how it looks on `develop`:

![image](https://github.com/user-attachments/assets/96cfb2df-d4ff-4451-8f40-f325ded75ea8)